src/Bicep.LangServer/Handlers/BicepDisableLinterRuleHandler.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Bicep.Core;
+using MediatR;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using OmniSharp.Extensions.JsonRpc;
+using OmniSharp.Extensions.LanguageServer.Protocol;
+using OmniSharp.Extensions.LanguageServer.Protocol.Workspace;
+
+namespace Bicep.LanguageServer.Handlers
+{
+    public class BicepDisableLinterRuleHandler : ExecuteTypedCommandHandlerBase<DocumentUri, string, string>
```

Comment:
>BicepDisableLinterRuleHandler [](http://example.com/codeflow?start=17&length=29)

Can we call it `BicepDisableLinterRuleCommandHandler` to make it more obvious that we're using a command to trigger this behavior?

Reply:
Sure, updated here:
https://github.com/Azure/bicep/commit/3e81e15f6c167ec9e6d16bd4a9fa056e4b3f3047

Thanks!